assets/scss/components/_wishlist.scss

```diff
@@ -0,0 +1,13 @@
+.c-wishlist {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+}
+
+.c-wishlist-add__button {
```

Comment:
@mgbelegu another place where the BEM seems to be a bit mixed. I think it this case it should be `c-wishlist__add-button` or simply `c-wishlist__button` to make it less explicit. You can always use the modifier to give more context like `c-wishlist__button--primary`